decision_service/include/err_constants.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+namespace reinforcement_learning { namespace error_code {
+  //success code
+  const int success = 0;
+
+  //error code
+  const int invalid_argument            = 1;
```

Comment:
>const int invalid_argument            = 1; [](start = 2, length = 42)

can we introduce some macro so that the error codes and strings are correctly mapped and we have some consistent mapping?

Reply:
Many of the errors are not mapped to static strings.  What did you have in mind?  can you provide an example?

---
In reply to: [193405817](https://github.com/JohnLangford/vowpal_wabbit/pull/1495#discussion_r193405817) [](ancestors = 193405817)